File: pset6/mario/mario.py
from cs50 import get_int

# Get height from user
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# Make the pyramid!
for i in range(n):
    print(" " * (n-i-1), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1))

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void print_space(int n);
void print_hash(int n);

int main(void)
{
    //First, ask the user for the height of the pyramid
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //Make the pyramid
    for (int i = 1; i <= n; i++)
    {
        print_space(n - i);
        print_hash(i);
        print_space(2);
        print_hash(i);
        printf("\n");
    }
}
//Method to print " " n times
void print_space(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf(" ");
    }
}
//Method to print a "#" n times
void print_hash(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("#");
    }
}